01-intro.Rmd

```diff
@@ -43,89 +69,83 @@ rm(re)
 The importance of reproducibility in scientific research should be obvious: if findings cannot be repeated, this casts doubt on the validity and truth of the conclusions drawn from them.
 Reproducibility is vital for *falsifiability*, a cornerstone of science [@popper_logic_1959].
 
-In applied, policy-relevant research areas such as road safety research, reproducibility is equally important:
-policy makers and the public want to have confidence that the evidence underlying key decisions is reliable.
-Policies based on results that nobody can reproduce are harder to defend than policies that are based on a clear evidence base that others, including members of the public and educators, can repeat.
+In applied policy-relevant research areas, such as road safety research, reproducibility is equally important: policy makers and the public want to have confidence that the evidence underlying key decisions is reliable. 
+Policies based on results that nobody can reproduce are harder to defend than policies that have a clear evidence base open for others to repeat, including members of the public and educators. 
 In transport planning, open and reproducible methods support more transparent and democratically accountable interventions [@lovelace_open_2020].
-Reproducibility leads to solid science which is conducive to effective policies.
+Reproducibility leads to solid science, which is conducive to effective policies.
 In the context of road safety research, this means that **reproducibility can save lives**.
 
 ## What is R?
 
 R is an open source programming language first developed by award-winning academic statisticians Dr [Ross Ihaka](https://en.wikipedia.org/wiki/Ross_Ihaka) and Professor [Robert Gentleman](https://en.wikipedia.org/wiki/Robert_Gentleman_(statistician)).
 Since its first release in 1995 and the release of version 1.0.0 in 2000, R has seen rapid uptake [@gentleman_statistical_2007; @ihaka_language_1996].
-As of September 2020, R is ranked as the 9^th^ most used programming language on the [TIOBE Index](https://www.tiobe.com/tiobe-index/), ahead of other languages for data processing such as SQL and MATLAB and behind *general purpose* languages such as C, Java and Python.
```

Comment:
Heads-up the ^th^ syntax was deliberate, always check the html version if in any doubt.